yellowbrick/yb_palettes.py

```diff
@@ -86,40 +86,58 @@ def as_hex(self):
 
 def color_palette(palette=None, n_colors=None, desat=None):
     """Return a list of colors defining a color palette.
-    Availible seaborn palette names:
-        accent, dark, paired, pastel, bold, muted
-    Availible seaborn palette names:
-        sns_deep, sns_muted, sns_bright, sns_pastel, sns_dark, sns_colorblind
-    Other options:
-        list of colors
     Calling this function with ``palette=None`` will return the current
     matplotlib color cycle.
     This function can also be used in a ``with`` statement to temporarily
     set the color cycle for a plot or set of plots.
-    Parameters
-    ----------
-    palette: None, string, or sequence, optional
+
+    :param palette:
         Name of palette or None to return current palette. If a sequence, input
         colors are used but possibly cycled and desaturated.
-    n_colors : int, optional
+
+        Available seaborn palette names from :py:mod:`seaborn.palettes` are:
```

Comment:
This *would* be an example of intersphinx but seaborn did not create docs for this module. This however is the syntax you would use and if sphinx can find a link to what you specify using intersphinx, it will link out to the external documentation.